target/imagebuilder/files/Makefile

```
@@ -237,6 +245,13 @@ ifeq ($(CONFIG_USE_APK),)
 			$(SCRIPT_DIR)/opkg-key add $(BUILD_KEY).pub \
 		) \
 	)
+else
+	$(if $(CONFIG_SIGNATURE_CHECK), \
+		$(if $(ADD_LOCAL_KEY), \
+			[ -d $(TARGET_DIR)/etc/opkg/keys/ ] || mkdir -p $(TARGET_DIR)/etc/opkg/keys/; \
```

```suggestion
			mkdir -p $(TARGET_DIR)/etc/opkg/keys/; \
```

No need to check if using `mkdir -p` as it will report success even if path already exists:

```
       -p, --parents
              no error if existing, make parent directories as needed, with their file modes unaffected by any -m option.
```

---

@KanjiMonster thanks for the check, I added this change to the other APK pr.